### PR TITLE
GE 21 1012 - Tujibebe & Chhaajaa Meta Tags

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

File: gem/templates/tanzania/base.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 <!--<![endif]-->
   <head>
     <meta charset="utf-8"/>
-    <title>{% block title %}{% if self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}{% endblock %}</title>
+    <title>Tujibebe | {% if self.seo_title %} {% if self.seo_title == "Main" %} Home {% else %}{{ self.seo_title }}{% endif %}{% else %} {% if self.title == "Main" %} Home {% else %}{{ self.title }}{% endif %}{% endif %}</title>
```
Comment (moh-moola) [inline gem/templates/tanzania/base.html]: @Mitso the seo page titles don't the get translated?

Comment (moh-moola) [reply gem/templates/tanzania/base.html]: if logic blocks have been changed to if blocks...

Comment (Mitso) [reply gem/templates/tanzania/base.html]: Thanks @moh-moola , the dynamic content get the current page translation from the CMS.

Comment (moh-moola) [reply gem/templates/tanzania/base.html]: @Mitso I think this should be spread across multiple lines, it makes it easier to understand and see logical errors

**Pull request merged**